lib/sidekiq_alive/worker.rb

```diff
@@ -1,18 +1,30 @@
+require "sidekiq/api"
 module SidekiqAlive
   class Worker
     include Sidekiq::Worker
-    sidekiq_options retry: false
+    sidekiq_options retry: false, queue: SidekiqAlive.config.queue_with_variant
```

Comment:
How do we ensure the config file is loaded before this file?.
In rails might work because is in the initializers folder.
any idea?

Reply:
We could move it to ENV